src/02-data-types.js

```diff
@@ -127,7 +143,9 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
+function isTruthy(val) {
+  return val ? true : false 
```

Comment:
remember we also have the !! operator here as well  - I actually prefer your ternary solution but `!!` works as well.